File: Programacion_Orientada_a_Objetos/CursoPOOUber/Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car{
    Map<String, ArrayList<String>> typeCarAcepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, Account driver, Map<String, ArrayList<String>> typeCarAcepted, ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAcepted = typeCarAcepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: Programacion_Orientada_a_Objetos/CursoPOOUber/PHP/uberblack.php
<?php
require_once("car.php");
class UberBlack extends Car
{
    public $typeCarAcepted;
    public $seatsMaterial;

    public function __construct(String $name, Account $driver, $typeCarAcepted, $seatsMaterial)
    {
        parent::__construct($license, $driver);
        $this->typeCarAcepted = $typeCarAcepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}
    
?>